source/minimessage.rst

```diff
@@ -331,5 +333,116 @@ The markdown parser supports the following markup:
    ``_italic_`` will be transformed into ``<italic>italic</italic>`` too
 * Underline:
    ``~~underline~~`` will be transformed into ``<underlined>underline</underlined>``
+* Obfuscated:
+   ``||obfuscated||`` will be transformed into ``<obfuscated>obfuscated</obfuscated>``
+
+However, this markup is a bit strange, but now its a too late to change, thats why we got:
+
+Markdown Flavors
+----------------
+
+What you saw above is the default/legacy favor. It will hopefully eventually be removed.
+
+To use different markdown flavors, you can use ``MiniMessage.withMarkdownFlavor(DiscordFlavor.get())`` or the Builder_.
+
+The discord flavor works like this: ``**bold**, *italic*, __underline__, ~~strikethrough~~, ||obfusctated||``
+
+The github flavor works like this: ``**bold**`, *italic*, ~~strikethrough~~``
+
+Additionally, you can implement your own markdown flavors. Take a look at the inbuild flavors for reference!
+
+Placeholder
+^^^^^^^^^^^^^^^^^^^
+
+MiniMessage provides two systems for placeholders. Depending on how you count. Could be 4 too.
+
+The easiest one is simple string replacements:
+``MiniMessage.get().parse("<gray>Hello <name>", "name", "MiniDigger")``
+
+As you can see, placeholders are defined like normal tags in the message, and resolve by a list of key value pairs (you can also pass a ``Map<String, String>`` here).
+
+These placeholders are resolved before any other tags in the message. This means, replacements can contain MiniMessage tags:
+ .. code:: java
+
+    String name = "MiniDigger";
+    String rank = "<red>[ADMIN]</red>"
+    Map<String, String> placeholders = Map.of("name", rank + name);
```

Comment:
Do we want to use API from Java >8 in docs? people are likely to get confused.

Reply:
Uuhi I didn't even notice that, I wrote that by hand, it's just to show that you can pass a map, but yeah, I guess it's better to remove